assemblies/assembly-planner-configuration.adoc

```diff
@@ -542,6 +542,145 @@ most appenders, including the console, cause congestion with `debug` and `trace`
 Switch to an async appender to avoid this problem or turn off `debug` logging.
 ====
 
+[[monitoring]]
+[id='monitoring-the-solver-proc_{context}']
+=== Monitoring the solver
+OptaPlanner exposes metrics through https://micrometer.io/[Micrometer], a metrics instrumentation library for Java applications. You can use Micrometer with popular monitoring systems to monitor the OptaPlanner solver.
+
+==== Configuring a Quarkus OptaPlanner application for Micrometer
+To configure your OptaPlanner Quarkus application to use Micrometer and a specified monitoring system, add the Micrometer dependency to the `pom.xml` file.
+
+.Prerequisites
+* You have a Quarkus OptaPlanner application.
+
+.Procedure
+. Add the following dependency to your application's `pom.xml` file where `<MONITORING_SYSTEM>` is a monitoring system supported by Micrometer:
```

Comment:
Need to be supported by Quarkus AND Micrometer. Prometheus is the only registry supported by Quarkus currently.